### ISSUE TITLE
Reimplementing links with JS

### ISSUE THREAD

**Issue by oliverjam** [fac20/week6-ELMP#49]
https://github.com/fac20/week6-ELMP/blob/ff7d6a031fef7ab9c7c6e36169b189b487cde2bd/public/index.js#L4-L10

These buttons should probably be anchor tags, since all they do is navigate to another page.

```html
<a href="signup-page">Sign up</a>
```

Using the right HTML elements is important for accessibility, amongst other things. Using normal links would also be simpler and would mean the page works even if your JS fails to load or throws an error.
